compiler/q-implant/src/QImplant.cpp

```diff
@@ -282,4 +285,38 @@ void QImplant::write(loco::Graph *g)
   }
 }
 
+void QImplant::forward_qparam(loco::Graph *g)
+{
+  std::set<luci::CircleOpcode> forwardable_opcode{
+    luci::CircleOpcode::RESHAPE, luci::CircleOpcode::SPLIT, luci::CircleOpcode::TRANSPOSE};
+
+  auto forwardable = [&forwardable_opcode](luci::CircleOpcode opcode) {
+    return forwardable_opcode.find(opcode) != forwardable_opcode.end();
+  };
+
+  for (auto node : loco::postorder_traversal(loco::output_nodes(g)))
```

Comment:
use `loco::active_nodes()`

Reply:
I found that `postorder_traversal` can give duplicate Node.

If I understand well, using `loco::active_nodes()` just intend to remove duplicated node, right?

`active_nodes` uses `set` to remove duplication, and I intend to process `in-order` from input
`set` orders elements, but I not sure about it will keep node order. (I think `set` will order by address of node, not sequence)

To remove duplication and keep node order, can I use `loco::postorder_traversal` and remove duplication with `set`?

Reply:
> I think set will order by address of node, not sequence

circle files are ordered in execution sequence, which nodes are created in that sequence which will have memory address of that sequence.
please look inside `loco::active_nodes()` source code.

Reply:
> nodes are created in that sequence which will have memory address of that sequence.

Never know about that node will be placed as execution sequence!
The address of node can guarantee it, so `loco::active_nodes()` would be great solution!